src/views/SystemList/Table.tsx

```diff
@@ -207,23 +274,67 @@ export const Table = ({
 
   rows.map(row => prepareRow(row));
 
+  if (
+    (isMySystems || tableType === 'my-systems') &&
+    loading &&
+    filteredSystems.length === 0
```

Comment:
Should this be a check for filteredSystems nullable instead of list length? Is it technically correct for an empty list to exist here? (but be considered as loaded)